README.md

```diff
@@ -221,6 +213,16 @@ just a few small guidelines you need to follow.
 [well-formed commit messages]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
 [pull request]: https://help.github.com/articles/creating-a-pull-request
 
+## Reduced order modelling mathLab's packages
+
+Below you can find a list of interesting and useful other available packages:
```

Comment:
Below you can find a list of useful reduced order modelling packages from SISSA mathLab group: